src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -6509,9 +6509,6 @@ namespace VariableSpeedCoils {
         state.dataVariableSpeedCoils->QSource *= PartLoadRatio;
         QWasteHeat *= PartLoadRatio;
 
-        //  Add power to global variable so power can be summed by parent object
-        state.dataHVACGlobal->DXElecCoolingPower = state.dataVariableSpeedCoils->Winput;
-
```

Comment:
Moved `state.dataHVACGlobal->DXElecCoolingPower` to `UpdateVarSpeedCoil()` function such the latest iteration value is used.